package.json

```diff
@@ -41,7 +41,8 @@
     "deep-equal": "^1.0.1",
     "es6-error": "^2.1.0",
     "hoist-non-react-statics": "^1.0.5",
-    "is-promise": "^2.1.0"
+    "is-promise": "^2.1.0",
+    "lodash.isequalwith": "^4.1.4"
```

Comment:
I did this in a separate commit. https://github.com/erikras/redux-form/commit/7107c888ada3f43e0f21276ecf2564edd831223c